css/index.css

```diff
@@ -1 +1,440 @@
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+main .middle__middle-img {
+	max-width: 876;
+	width: 100%;
+}
+
+main .bottom {
+	width: 100%;
+	margin: 31px 0 31px 0;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	flex-wrap: wrap;
+}
+
+main .bottom__left-box {
+	width: 33.3%;
+	padding-right: 2%;
+	justify-content: flex-start;
+}
+
+main .bottom__mid-box {
+	width: 33.3%;
+	padding-right: 2%;
+	justify-content: center;
+
+}
+main .bottom__right-box {
+	width: 33.3%;
+	padding-right: 2%;
+	justify-content: flex-end;
+}
+
+main .bottom p {
+	font-size: 1.62rem;
+	line-height: 1.17;
+}
+
+/* ----------------------------------- desktop contact  */
+.contact {
+	display: flex;
+	padding-top: 23.5px;
+	flex-direction: column;
+	justify-content: flex-start;
+	align-content: space-around;
+	border-top: 1px solid black;
+}
+
+.contact h2 {
+	margin: 13px 0 2px 0;
+}
+
+.contact p {
+	margin: 14px 0 3px 0;
+	font-size: 1.6rem;
+	line-height: 0.9;
+}
+
+/* --------------------------------- desktop footer */
+footer {
+	width: 100%;
+	padding-top: 40px;
+	display: flex;
+	justify-content: center;
+	font-size: 1.6rem;
+}
+
+/* =========================== TABLET ========================================= */
+
+@media (max-width: 800px) {
+	 .container {
+		width: 100%;
+		margin: 0 auto;
+		display: flex;
+        flex-direction: column;
+        flex-wrap: wrap;
+		align-items: center;
+	}
+/* --------------------------------------- tablet nav  */
+	nav {
+		width: 92%;
+		margin: 0 2% 0 0;
+		display: flex;
+		flex-direction: column-reverse;
+        justify-content: center;
+		align-items: center;
+	}
+
+	.nav__nav-items {
+        width: 100%;
+        margin-bottom: 24.5px;
+		padding: 20px 0.5% 0 1%;
+		display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+		align-items: center;
+	}
+
+	.nav__nav-items a {
+        font-size: 1.6rem;
+		line-height: 1.2;
+    }
+	.nav__logo {
+		max-width: 183;
+		width: 25.5%;
+		margin: 25px 30px 0 9%;
+	}
+
+	/* ----------------------------------- tablet header */
+	header {
+		width: 90%;
+		margin: 0 3.5% 0 0;
+		display: flex;
+        justify-content: center;
+        border: none;
+	}
+
+	.header__getting-started {
+		width: 90%;
+		margin: 0 auto;
+		padding: 0 auto;
+		display: flex;
+		flex-direction: column;
+		justify-content: flex-start;
+		}
+
+	.header__getting-started_motto {
+		width: 100%;
+		margin: 0 auto 7px 1%;
+		padding: 0 auto;
+		display: flex;
+		flex-direction: column;
+		justify-content: flex-start;
+	}
+
+	.header__start {
+		width: 28%;
+		font-size: 1.8rem;
+        margin: 0 auto;
+        margin-top: 1.6px;
+        margin-left: 36.8%;
+		padding: 0 auto 6px auto;
+		flex-direction: row;
+		justify-content: center;
+		text-align: center;
+	}
+
+	.header__snippet {
+		display: none;
+	}
+
+/* ---------------------------------------- tablet main */
+	main {
+		width: 97.4%;
+        margin: 0 auto;
```

Comment:
I'm wondering if you use this varied indentation to visually differentiate between categories of styles (eg. margins together, flex-related styles together).